src/services/httpClient.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { Observable } from 'rxjs';
+import { AuthService } from './authService';
+import Config from 'app.config';
+import { AjaxError, RetryableAjaxError } from './models';
+
+/**
+ * A class of static methods for creating ajax requests
+ */
+export class HttpClient {
+
+  /**
+   * Constructs a GET ajax request
+   *
+   * @param {string} url The url path to the make the request to
+   */
+  static get(url, options = {}, withAuth = true) {
+    return HttpClient.ajax(url, { ...options, method: 'GET' }, withAuth);
```

Comment:
As discussed, we can make them an enum but since these are never called explicitly I don't think it is worth it. The methods themselves essentially act as enums.